### PR TITLE
Update for breaking changes

### PR DESCRIPTION
**Pull request by FricativeMelon**

### CODE REVIEW

**Comment by FricativeMelon** [comment]
Ignore my changes to eventData.json if possible

File: RaisedGardenBeds/ModEntry.cs [hunk 1]
```diff
@@ -12,8 +12,23 @@ namespace RaisedGardenBeds
...
+
+        internal static readonly string LocalAssetPath = "assets";
+
+        internal static readonly string LocalEndOfNightSpritesPath = Path.Combine(LocalAssetPath, "endOfNightSprites.png");
+        internal static readonly string LocalEventDataPath = Path.Combine(LocalAssetPath, "eventData.json");
+
+        internal static readonly string ContentPackPath = Path.Combine(LocalAssetPath, "ContentPack");
+
+        // common
+        internal static ModEntry Instance;
```
Comment (b-b-blueberry) [inline RaisedGardenBeds/ModEntry.cs]: Did you mean to copy these fields from `AssetManager`? They appear to be defined in both files.

File: RaisedGardenBeds/AssetManager.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 namespace RaisedGardenBeds
 {
-	public class AssetManager : IAssetLoader, IAssetEditor
+	public class AssetManager
 	{
 		private readonly IModHelper _helper;
```
Comment (b-b-blueberry) [inline RaisedGardenBeds/AssetManager.cs:13]: Existing fields also defined in `ModEntry`

**Review by b-b-blueberry — CHANGES_REQUESTED** [review]
Thanks for submitting, and sorry for taking so long to check in! It all looks good except for some fields being copied from `AssetManager` into `ModEntry`, maybe some local changes that were meant to be removed? Push up a new commit with a fix for that and it should be good to go for a Nexus release!

**Comment by FricativeMelon** [comment]
Made changes!

**Comment by FricativeMelon** [comment]
Wait it has errors in game, let me take a look!

**Comment by FricativeMelon** [comment]
False alarm, everything seems to be working fine!

**Review by b-b-blueberry — APPROVED** [review]
Looks perfect, thanks again!

**Pull request merged**